src/views/shared/SecurityBeacon.js

```diff
@@ -45,7 +45,14 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     }
   }
 
-  function antiPhishingMessage (image, host, shown) {
+  function antiPhishingMessage (image, host) {
+    function showQtip() {
+      if(image.is(':visible')){
+        image.qtip('show');
+      }
+    }
+    $(window).on('resize.securityBeaconQtip', _.debounce(showQtip, 300));
```

Comment:
True. It's just that the function is basically doing a one-line call, so I didn't want to add an extra function for something as simple. But either works, so up to you.